Description: annotate source for ESLint
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2020-06-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/NodeList.es6.js
+++ b/lib/NodeList.es6.js
@@ -1,3 +1,4 @@
+/* eslint-env es6 */
 /* jshint esversion: 6 */
 "use strict";
 
